### PR TITLE
Create policy for afterburn

### PR DESCRIPTION
**Pull request by 5umm3r15**
Copr build:

dnf copr enable nknazeko/afterburn-selinux-fedora
dnf update selinux-policy

### CODE REVIEW

**Comment by 5umm3r15** [comment]
Ready for review @zpytela

File: policy/modules/contrib/afterburn.if [hunk 1]
```diff
@@ -0,0 +1,40 @@
+
```
Comment (zpytela) [inline policy/modules/contrib/afterburn.if]: Initial blank line

File: policy/modules/contrib/afterburn.if [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+## <summary>policy for afterburn</summary>
+
+########################################
+## <summary>
+##	Execute afterburn_exec_t in the afterburn domain.
```
Comment (zpytela) [inline policy/modules/contrib/afterburn.if]: rather just execute afterburn

File: policy/modules/contrib/afterburn.te [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+kernel_read_all_proc(afterburn_t)
+
+corenet_tcp_connect_http_port(afterburn_t)
+
+domain_use_interactive_fds(afterburn_t)
+
+files_read_etc_files(afterburn_t)
+
+auth_use_nsswitch(afterburn_t)
```
Comment (zpytela) [inline policy/modules/contrib/afterburn.te]: auth, miscfiles, and sysnet should be in an optional block each

File: policy/modules/contrib/afterburn.fc [hunk 4]
```diff
@@ -0,0 +1,7 @@
+/usr/bin/afterburn						--	gen_context(system_u:object_r:afterburn_exec_t,s0)
+
+/usr/lib/systemd/system/afterburn-checkin\.service		--	gen_context(system_u:object_r:afterburn_unit_file_t,s0)
+/usr/lib/systemd/system/afterburn-firstboot-checkin\.service	--	gen_context(system_u:object_r:afterburn_unit_file_t,s0)
+/usr/lib/systemd/system/afterburn-sshkeys\.target		--	gen_context(system_u:object_r:afterburn_unit_file_t,s0)
+/usr/lib/systemd/system/afterburn-sshkeys@\.service		--	gen_context(system_u:object_r:afterburn_unit_file_t,s0)
+/usr/lib/systemd/system/afterburn\.service			--	gen_context(system_u:object_r:afterburn_unit_file_t,s0)
```
Comment (zpytela) [inline policy/modules/contrib/afterburn.fc]: Can this be a single-line entry using a regexp or was there any conflict?

File: policy/modules/contrib/afterburn.if (hunk 1 above)
Comment (5umm3r15) [reply policy/modules/contrib/afterburn.if]: Fixed, thanks

File: policy/modules/contrib/afterburn.fc (hunk 4 above)
Comment (5umm3r15) [reply policy/modules/contrib/afterburn.fc]: You are right, fixed

**Comment by zpytela** [comment]
Merging, thank you.

**Pull request merged**